<launch>

  <param name="use_sim_time" value="false"/>

  <include file="$(find smb_description)/launch/debug.launch">
    <!--<arg name="visualize" value="1"/>-->
  </include>

  <arg name="object_topic" default='/objects'/>
  <arg name="visualize_all" default='False'/>
  <arg name="only_BB" default='True'/>

  <node name="object_visualization" pkg="object_visualization" type="node.py" output="screen">
    <param name="object_topic" value='$(arg object_topic)'/>
    <param name="out_image_pub_topic" value='/versavis/cam0/objects'/>
    <param name="visualize_all" value='$(arg visualize_all)'/>
    <param name="only_BB" value='$(arg only_BB)'/>
    <param name="map_frame" value='map'/>

  </node>
  
</launch>
